extension/fts/fts_indexing.cpp

```diff
@@ -0,0 +1,311 @@
+#include "fts_indexing.hpp"
+
+#include "duckdb/main/connection.hpp"
+#include "duckdb/catalog/catalog_entry/table_catalog_entry.hpp"
+#include "duckdb/common/exception.hpp"
+#include "duckdb/common/string_util.hpp"
+#include "duckdb/parser/qualified_name.hpp"
+
+namespace duckdb {
+
+// weird way to have decently readable SQL in here
+#define SQL(...) #__VA_ARGS__
```

Comment:
Can't you use C++11 [raw string literals](https://en.cppreference.com/w/cpp/language/string_literal)? e.g. 

```cpp
const char *sql = R"(

SELECT *
FROM table
WHERE i=3

");
```

Reply:
I did not know about these. Looks much better. Done!